drivers/SmartThings/zigbee-power-meter/src/test/test_zigbee_power_meter_consumption_report.lua

```diff
@@ -100,50 +100,66 @@ test.register_message_test(
   }
 )
 
+-- TODO should this test be removed, or should the driver be fixed so it passes?
```

Comment:
The [device](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/494974b8c37e1ab3274001b8d43ebe2e7083ca80/drivers/SmartThings/zigbee-power-meter/fingerprints.yml#L2-L5) being tested in this test uses the [ezex](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/494974b8c37e1ab3274001b8d43ebe2e7083ca80/drivers/SmartThings/zigbee-power-meter/src/ezex/init.lua#L24) subdriver. It seems like this is another case where the subdriver writer expected default behavior to exist, in this case the default configured attributes, but it doesn't since the subdriver doesn't specify them. It's not clear to me whether the subdriver should be overriding the configure handler, or even what the configure functionality for this device should be; no attributes are being configured on this device.

Reply:
Actually, it looks like the mock device doesn't specify any server clusters which is why no attributes are configured.